components/compliance-service/integration_test/stats_server_read_summary_test.go

```diff
@@ -0,0 +1,323 @@
+
+			//controls summary
+			expectedPassedCnt: 2,
+		},
+		{
+			description:     "Projects: user has access to some projects with reports",
+			allowedProjects: []string{"project1", "project2"},
+			query:           octoberTwentyFifthQuery,
+
+			//report summary
+			expectedEnvironmentCnt: 4,
+			expectedPlatformCnt:    4,
+			expectedProfileCnt:     4,
+			expectedNodeCnt:        4,
+			expectedStatus:         "passed",
+
+			//nodes summary
+			expectedCompliantCnt:    4,
+			expectedNonCompliantCnt: 0,
+
+			//controls summary
+			expectedPassedCnt: 4,
+		},
+		{
+			description:     "Projects: user has access to projects without reports",
+			allowedProjects: []string{"project4", "project5"},
+			query:           octoberTwentyFifthQuery,
+
+			//report summary
+			expectedEnvironmentCnt: 0,
+			expectedPlatformCnt:    0,
+			expectedProfileCnt:     0,
+			expectedNodeCnt:        0,
+			expectedStatus:         "unknown",
+
+			//nodes summary
+			expectedCompliantCnt:    0,
+			expectedNonCompliantCnt: 0,
+
+			//controls summary
+			expectedPassedCnt: 0,
+		},
+		{
+			description:     "Projects: user has access to one project with reports and unassigned reports",
+			allowedProjects: []string{"project1", authzConstants.UnassignedProjectID},
+			query:           octoberTwentyFifthQuery,
+
+			//report summary
+			expectedEnvironmentCnt: 3,
+			expectedPlatformCnt:    3,
+			expectedProfileCnt:     3,
+			expectedNodeCnt:        3,
+			expectedStatus:         "passed",
+
+			//nodes summary
+			expectedCompliantCnt:    3,
+			expectedNonCompliantCnt: 0,
+
+			//controls summary
+			expectedPassedCnt: 3,
+		},
+		{
+			description:     "Projects: user has access to some projects with reports and unassigned reports",
+			allowedProjects: []string{"project1", "project2", authzConstants.UnassignedProjectID},
+			query:           octoberTwentyFifthQuery,
+
+			//report summary
+			expectedEnvironmentCnt: 5,
+			expectedPlatformCnt:    5,
+			expectedProfileCnt:     5,
+			expectedNodeCnt:        5,
+			expectedStatus:         "passed",
+
+			//nodes summary
+			expectedCompliantCnt:    5,
+			expectedNonCompliantCnt: 0,
+
+			//controls summary
+			expectedPassedCnt: 5,
+		},
+		{
+			description:     "Projects: user has access to projects without reports and unassigned reports",
+			allowedProjects: []string{"project4", "project5", authzConstants.UnassignedProjectID},
+			query:           octoberTwentyFifthQuery,
+
+			//report summary
+			expectedEnvironmentCnt: 1,
+			expectedPlatformCnt:    1,
+			expectedProfileCnt:     1,
+			expectedNodeCnt:        1,
+			expectedStatus:         "passed",
+
+			//nodes summary
+			expectedCompliantCnt:    1,
+			expectedNonCompliantCnt: 0,
+
+			//controls summary
+			expectedPassedCnt: 1,
+		},
+		{
+			description:     "Projects: user has access to unassigned reports",
+			allowedProjects: []string{authzConstants.UnassignedProjectID},
+			query:           octoberTwentyFifthQuery,
```

Comment:
If `query` is the same for all the test cases, it could be moved into the test definition below.

Reply:
fixed.. thanks